integration-tests/wallet-empty-unrevealed-implicit-account.spec.ts

```diff
@@ -31,7 +34,7 @@ CONFIGS().forEach(({ lib, rpc, setup, createAddress }) => {
       const op3 = await LocalTez.wallet.transfer({ to: await Tezos.signer.publicKeyHash(), mutez: true, amount: maxAmount}).send();
       await op3.confirmation();
 
-     // expect((await Tezos.tz.getBalance(await LocalTez.signer.publicKeyHash())).toString()).toEqual("0")
+      expect((await Tezos.tz.getBalance(await LocalTez.signer.publicKeyHash())).toString()).toEqual("0")
```

Comment:
Somethings seems to be wrong with the test. It remains 40 instead of being drained.
This will need investigation.

```
 FAIL ./wallet-empty-unrevealed-implicit-account.spec.ts (135.967 s)
  ● Test emptying an unrevealed implicit wallet account using: https://api.tez.ie/rpc/edonet › creates an unrevealed implicit wallet account, funds it, and empties it

    expect(received).toEqual(expected) // deep equality

    Expected: "0"
    Received: "40"
```

Reply:
why does the test have these lines?
    it('does nothing', async (done) => {
      done();
    })